src/qt/pivx/masternodewizarddialog.cpp

```diff
@@ -94,7 +94,7 @@ MasterNodeWizardDialog::MasterNodeWizardDialog(WalletModel *model, QWidget *pare
     setCssProperty(ui->pushButtonSkip, "ic-close");
 
     connect(ui->pushButtonSkip, &QPushButton::clicked, this, &MasterNodeWizardDialog::close);
-    connect(ui->btnNext, &QPushButton::clicked, this, &MasterNodeWizardDialog::onNextClicked);
+    connect(ui->btnNext, &QPushButton::clicked, this, &MasterNodeWizardDialog::accept);
```

Comment:
I agree it makes the code a little bit less readable.
Since we want the "go to next" action with the ENTER key here, this is basically an hack to use directly the inherited implementation of `keyPressEvent`from `FocusedDialog`.
If preferred, in alternative, I can restore the name `onNextClicked` here and override `keyPressEvent`. Little bit of code duplication, but better readability. Either way is fine for me.

Reply:
yeah ok, no problem, make sense now.